lib/HoplonMessages.asn1

```diff
@@ -0,0 +1,48 @@
+HoplonMessages DEFINITIONS AUTOMATIC TAGS ::= BEGIN
+
+EXPORTS Person, Package, Verdict, Audit, SignedAudit;
+
+Person ::= [19] SEQUENCE {
+
+        name UTF8String,
+        location INTEGER {home(0),field(1),roving(2)},
+        age INTEGER OPTIONAL }
```

Comment:
I still like the the validation on the DER structure with Person, let's just remove it before the release.